axelrod/random_.py

```diff
@@ -3,6 +3,12 @@
 from axelrod import Actions
 
 
+choices = {
```

Comment:
Again: completely stylistic preference so just ignore me but FWIW, this could also be done as a list:

choices = [Actions.D, Actions.C]
The lookup below just becomes indexing. (Interestingly, just checked and it still works with the boolean. Neat.)

(Whether or not it should is another question.)